.tekton/scanner-v4-pipeline.yaml

```diff
@@ -101,7 +101,7 @@ spec:
     value: $(tasks.clone-repository.results.commit)
   - description: ""
     name: JAVA_COMMUNITY_DEPENDENCIES
-    value: $(tasks.build-container.results.JAVA_COMMUNITY_DEPENDENCIES)
+    value: $(tasks.build-container-amd64.results.JAVA_COMMUNITY_DEPENDENCIES)
 
```

Comment:
Please update `init` task comment. Similar to https://github.com/stackrox/stackrox/pull/12042